components/autofill/android/build.gradle

```diff
@@ -102,9 +102,8 @@ dependencies {
 // We do the uniffi binding generation here:
 android.libraryVariants.all { variant ->
     def t = tasks.register("generate${variant.name.capitalize()}UniffiBindings", Exec) {
-        workingDir "${project.projectDir}"
-        // Runs the bindings generation, note that you must have uniffi-bindgen installed and in your PATH environment variable
-        commandLine 'uniffi-bindgen', 'generate', '../src/autofill.udl', '--language', 'kotlin', '--out-dir', "${buildDir}/generated/source/uniffi/${variant.name}/java"
+        workingDir project.rootDir
+        commandLine 'cargo', 'uniffi-bindgen', 'generate', "${project.projectDir}/../src/autofill.udl", '--language', 'kotlin', '--out-dir', "${buildDir}/generated/source/uniffi/${variant.name}/java"
```

Comment:
Unfortunately `cargo uniffi-bindgen` doesn't work within a crate directory, only in the workspace root. We can still call it from within gradle in a reasonable fashion though.